packages/strapi-generate-new/lib/before.js

```diff
@@ -207,7 +210,14 @@ module.exports = (scope, cb) => {
               });
           }),
           new Promise(resolve => {
-            let cmd = `npm install --prefix "${scope.tmpPath}" ${scope.client.connector}@alpha`;
+            const isStrapiInstalledWithNPM = packageManager.isStrapiInstalledWithNPM();
+            let packageCmd = isStrapiInstalledWithNPM ? `npm install --prefix "${scope.tmpPath}" ${scope.client.connector}@alpha` : packageManager.commands('install --prefix', scope.tmpPath);
```

Comment:
Write a command for this one? Also, the first command installs a specific package with npm and it seems to install nothing with yarn?

```js
let packageCmd = packageManager.commands('install --prefix', scope.tmpPath, scope.client.connector);
```

Reply:
We don't need to pass the connector since the `packageManager` will return either `npm install --prefix ${path}` or `yarn --cwd ${path} add` and we add the connector to install in the next line.